GameData/ROCapsules/PartConfigs/GeminiBDB/GeminiHSLunar.cfg

```diff
@@ -0,0 +1,74 @@
+PART
+{
+	name = ROC-GeminiHSLunarBDB
+	module = Part
+	author = CobaltWolf, capkirk
+
+	RSSROConfig = True
+
+	MODEL
+	{
+		model = ROCapsules/Assets/BDB/Gemini/bluedog_Gemini_Heatshield_1p5m
+		rotation = 0, 180, 0
+	}
+
+	rescaleFactor = 1.524
+	node_stack_bottom = 0.0, 0.075149, 0.0, 0.0, -1.0, 0.0, 1
+	node_stack_top = 0.0, 0.075149, 0.0, 0.0, 1.0, 0.0, 1
+	CoPOffset = 0.0, 1.0, 0.0
+	CoLOffset = 0.0, -0.15, 0.0
+
+	TechRequired = flightControl
+	entryCost = 1000
+	cost = 360
+	category = Thermal
+	subcategory = 0
+	title = Gemini Lunar Heatshield
+	manufacturer = McDonnell Aircraft
+	description = Improved, lunar rated heatshield for the Gemini Capsule. Can be switched to the Gemini B configuration with a hatch
+	attachRules = 1,0,1,0,0
+	mass = 0.050
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag = 0.2
+	angularDrag = 2
+	breakingForce = 72
+	breakingTorque = 72
+	crashTolerance = 18
+	maxTemp = 1500
+	skinMaxTemp = 2600
+	skinSkinConductionMult = 0.1
+	fuelCrossFeed = False
```

Comment:
Note that maxTemp, skinMaxTemp, and mass will be set because of the heatshield tags below.
Unsure if you really want skinSkinConductionMult to be 0.1. See above comment.